routes/users/users-router.js

```diff
@@ -36,11 +36,12 @@ router.get("/:id", async (req, res) => {
 //endpoint to create a new user
 router.post("/", async (req, res) => {
   try {
-    let user = await Users.getUserBySubId(req.body.sub_id);
+    // let user = await Users.getUserBySubId(req.body.sub_id);
+    let user = await Users.getUserByEmail(req.body.email);
     if (!user) {
       //the next two lines create a user and then return newly created user with sub_id
```

Comment:
Change this comment to be `the next two lines create a user and then return newly created user with email`